requirements.txt

```diff
@@ -1,3 +1,7 @@
 lxml>=3.6.0
 future>=0.15.2
 wheel>=0.37.1
+
+# dev dependencies:
+pytest
```

Comment:
Should we add these dependencies for everyone on every install of pymavlink?

The vast majority of people won't want to run the test suite.

Perhaps we should add this as a requirements.txt in the `snapshottests` directory?

Reply:
> Should we add these dependencies for everyone on every install of pymavlink?

No, and this PR doesn't. When you `pip install` pymavlink, you *don't* get the stuff in requirements.txt. You get the (totally unversioned :shrug:) dependencies in `setup.py`.

Reply:
@peterbarker Response please.

Reply:
@peterbarker Ping.

Reply:
Cool, thanks.